Mage.Sets/src/mage/cards/r/RooftopBypass.java

```diff
@@ -0,0 +1,36 @@
+package mage.cards.r;
+
+import java.util.UUID;
+
+import mage.abilities.common.DealCombatDamageControlledTriggeredAbility;
+import mage.abilities.effects.common.CreateTokenEffect;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.filter.StaticFilters;
+import mage.game.permanent.token.AssassinToken3;
+
+/**
+ *
+ * @author grimreap124
+ */
+public final class RooftopBypass extends CardImpl {
+
+    public RooftopBypass(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.ENCHANTMENT}, "{1}{U}{B}");
+        
+
+        // Whenever one or more nontoken creatures you control deal combat damage to a player, create a 1/1 black Assassin creature token with menace.
+        this.addAbility(new DealCombatDamageControlledTriggeredAbility(new CreateTokenEffect(new AssassinToken3()),
```

Comment:
we already have `AssassinMenaceToken`

Reply:
also you need to pass a filter into the triggered ability

Reply:
I'm confused as to what you mean by passing a filter? I believe I am StaticFilters.FILTER_CONTROLLED_CREATURES_NON_TOKEN?

Reply:
yeah I just somehow missed that the filter was on the next line, you're good